pyccel/codegen/pipeline.py

```diff
@@ -36,6 +36,11 @@
     "c" : ".c",
     "fortran": ".f90",
 }
+# map language to its object file extension
+lang_obj_ext_dict = {
+    "c" : ".o",
+    "fortran": ".mod",
+}
```

Comment:
Maybe the names are wrong here. A Fortran `.mod` file contains the kind of information that one finds in a C header file, and is definitely not an object file. Fortran object files normally end with `.o` just like in C.

Reply:
Hmm, yes I was hoping to identify if the file had been compiled with the correct language, in case there were fortran and c versions of a file with the sam name, but you are correct this is not possible

Reply:
I have removed this